keras2onnx/ke2onnx/upsample.py

```diff
@@ -29,6 +29,12 @@ def convert_keras_upsample(scope, operator, container, n_dims):
     else:
         raise ValueError('Unsupported dimension %s when converting Keras Upsampling layer' % n_dims)
 
+    mode = 'nearest'
+    if n_dims == 2:
+        mode = op.interpolation
+        if mode == 'bilinear':
+            mode = 'linear'
```

Comment:
mode='linear' if mode.endswith('linear') ? it can do 1-d, 2-d, 3-d.

Reply:
Agreed.  I've switched to using endswith() to generalize it a bit.